data-protocols/ids/ids-api-multipart-endpoint-v1/src/main/java/org/eclipse/dataspaceconnector/ids/api/multipart/handler/DescriptionRequestHandler.java

```diff
@@ -88,19 +91,16 @@ public boolean canHandle(@NotNull MultipartRequest multipartRequest) {
         // Get ID of requested element
         var requestedElement = IdsId.from(message.getRequestedElement());
 
-        //TODO: IDS REFACTORING: this should be a named property of the message object
-        // extract paging information, default to 0 ... Integer.MAX_VALUE
-        var from = getInt(message, Range.FROM, 0);
-        var to = getInt(message, Range.TO, Integer.MAX_VALUE);
-        var range = new Range(from, to);
+        var range = getRange(message);
```

Comment:
As per my comments in the DR-PR: instead of separately transmitting the `range` and `filter`, I think it would be better to just transmit a JSON serialized `QuerySpec`. That would future-proof us in case we ever want to transmit sorting parameters as well.

Reply:
Done.